src/calculateRentalCost.js

```diff
@@ -28,7 +28,12 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 20;
+  const superDiscount = 50;
+  const price = 40;
+
+  return days < 3 ? days * price : days > 6
+    ? days * price - superDiscount : days * price - basicDiscount;
```

Comment:
Don't repeat yourself: create variable for days * price value
It's better to use if else statement instead on nesting with ternary operators